assets/chatglm.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- type: model
+  name: ChatGLM
+  organization: ChatGLM
+  description: ChatGLM is a Chinese-English language model with question and answer
+    and dialogue functions, and is aimed at a Chinese audience.
+  created_date: 2023-03-14
+  url: https://chatglm.cn/blog
+  model_card: ''
+  modality: text
+  analysis: Performance evaluated on English and Chinese language benchmark tests.
+  size: 6B parameters
+  dependencies: [GLM-130B, ChatGPT]
```

Comment:
I don't think this is right?

Reply:
Agreed, that was a translation error on my part. ChatGPT is not a dependency of ChatGLM.